lib/routes/account.js

```diff
@@ -235,6 +237,12 @@ module.exports = function (
             }).catch(function (err) {
               log.error({ op: 'mailer.sendVerifyCode.1', err: err })
             })
+
+            verificationReminder.create({
```

Comment:
Wonder if it would be better to only create a verification reminder if `mailer.sendVerifyCode` succeeds?